src/js/components/AdminPanels.tsx

```diff
@@ -188,7 +215,7 @@ export function DropDownMenuContainer({
         updateRedisStatus("false")
         setThisMenu((prevMenu) => ({
           ...prevMenu,
-          selectedItem: null,
+          selectedItem: undefined,
```

Comment:
Why this change from `null` to `undefined`? (It might be related with one of my previous comments).